iib/workers/tasks/legacy.py

```diff
@@ -36,8 +36,13 @@ def export_legacy_packages(packages, request_id, rebuilt_index_image, cnr_token,
             _verify_package_info(package_dir, rebuilt_index_image)
             _zip_package(package_dir)
             _push_package_manifest(package_dir, cnr_token, organization)
-
-    set_request_state(request_id, 'in_progress', 'Back ported packages successfully pushed to OMPS')
+    payload = {
+        'index_image': rebuilt_index_image,
+        'packages': packages,
+        'state': 'completed',
```

Comment:
This is an invalid state. It should be `complete` and not `completed`. The valid states are defined in `iib.web.models::RequestStateMapping`.

Reply:
Oops, yes I realized that my dev-env didn't test this part to raise errors